libpod/runtime.go

```diff
@@ -385,6 +385,26 @@ func NewRuntime(options ...RuntimeOption) (runtime *Runtime, err error) {
 		if _, err := toml.Decode(string(contents), runtime.config); err != nil {
 			return nil, errors.Wrapf(err, "error decoding configuration file %s", configPath)
 		}
+	} else if rootless.IsRootless() {
+		// If the configuration file was not found but we are running in rootless, a subset of the
+		// global config file is used.
+		for _, path := range []string{OverrideConfigPath, ConfigPath} {
+			contents, err := ioutil.ReadFile(OverrideConfigPath)
+			if err != nil {
+				// Ignore any error, the file might not be readable by us.
+				continue
+			}
+			tmpConfig := new(RuntimeConfig)
+			if _, err := toml.Decode(string(contents), tmpConfig); err != nil {
+				return nil, errors.Wrapf(err, "error decoding configuration file %s", path)
+			}
+
+			// Cherry pick the settings we want from the global configuration
+			runtime.config.ConmonPath = tmpConfig.ConmonPath
```

Comment:
Shouldn't we make sure these are not set, and only set them.

If I set the ConmonPath in my local config isn't this an issue.

Reply:
We are in this code branch only when `~/.config/containers/libpod.conf` doesn't exist.

The only place where we could set them is from the default configuration we create.

Reply:
Ok then LGTM